### PR TITLE
Update Polish translation

### PR DESCRIPTION
**Pull request by zimnyd**

### CODE REVIEW

File: po/pl.po [hunk 1]
```diff
@@ -8,67 +8,69 @@ msgstr ""
 "Project-Id-Version: elementary-community\n"
 "Report-Msgid-Bugs-To: \n"
 "POT-Creation-Date: 2021-08-05 22:16+0900\n"
-"PO-Revision-Date: 2015-01-14 12:48+0000\n"
+"PO-Revision-Date: 2021-08-21 09:33+0200\n"
 "Last-Translator: Patryk Strugacz <Unknown>\n"
```
Comment (ryonakano) [inline po/pl.po:12]: If you will, would you write your name (and your email address)? It's not necessary to be your real name; you can use your nickname like username on GitHub. And if you don't want to expose your email address, it's OK to add only your name.

This is just for credit you—that you edited this translation file and the translations are copyrighted to you.

**Review by ryonakano — DISMISSED** [review]
Hi, thanks for your translation! LGTM 👍🏻

Leaving just one optional comment.

File: po/pl.po (hunk 1 above)
Comment (zimnyd) [reply po/pl.po]: I just forget to change the credits info.

**Review by ryonakano — APPROVED** [review]
Thank you very much!

**Pull request merged**